earthpy/spatial.py

```diff
@@ -369,13 +369,13 @@ def crop_image(raster, geoms, all_touched=True):
         >>> from earthpy.io import path_to_example
         >>> # Clip an RGB image to the extent of Rocky Mountain National Park
         >>> rmnp = gpd.read_file(path_to_example("rmnp.shp"))
-        >>> with rio.open(path_to_example("rmnp-rgb.tif")) as raster:
-        ...     src_image = raster.read()
-        ...     out_image, out_meta = es.crop_image(raster, rmnp)
+        >>> with rio.open(path_to_example("rmnp-rgb.tif")) as src:
+        ...     in_image = src.read()
+        ...     out_image, out_meta = es.crop_image(src, rmnp)
+        >>> in_image.shape
```

Comment:
```suggestion
        >>> src_raster.shape[1:3]
```
We can just select the last two elements in the return tuple to return just the dimensions of the array. This will avoid needing a numpy array and thus the read() statement can go away!